package com.serviloc.fichiers.infrastructure.persistence;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;
import java.util.UUID;

@Repository
public interface FileMetadataJpaRepository extends JpaRepository<FileMetadataJpaEntity, UUID> {

    List<FileMetadataJpaEntity> findAllByIdIn(List<UUID> ids);
}
